bulk-scan/values.yaml

```diff
@@ -61,7 +61,41 @@ bulk-scan-orchestrator:
       SB_CONN_STRING:
         secretRef: servicebus-secret-namespace-{{ .Release.Name }}-sb
         key: connectionString
+      IDAM_CLIENT_SECRET:
+        secretRef: bsp-orchestrator-creds
+        key: idam-client-secret
+      IDAM_USERS_BULKSCAN_USERNAME:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-bulkscan-username
+      IDAM_USERS_BULKSCAN_PASSWORD:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-bulkscan-password
+      IDAM_USERS_CMC_USERNAME:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-cmc-username
+      IDAM_USERS_CMC_PASSWORD:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-cmc-password
+      IDAM_USERS_DIV_USERNAME:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-div-username
+      IDAM_USERS_DIV_PASSWORD:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-div-password
+      IDAM_USERS_PROBATE_USERNAME:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-probate-username
+      IDAM_USERS_PROBATE_PASSWORD:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-probate-password
+      IDAM_USERS_SSCS_USERNAME:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-sscs-username
+      IDAM_USERS_SSCS_PASSWORD:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-sscs-password
     environment:
+      S2S_SECRET: AAAAAAAAAAAAAAAA
```

Comment:
???

Reply:
Worth adding a comment explaining this value

Reply:
idam.s2s-auth.secret
s2s runs is in the pod.

Reply:
it is used throughout the config

Reply:
is it needed for those users though?

Reply:
By adding a comment I meant a comment in code. Or maybe we could reference a global value here?

Reply:
I think when we run the deployment these values are not using. after the flux deployment and test we will see . we can add all these to read.me after we run in demo and sure these values are correct.

Reply:
> is it needed for those users though?

I think no , but chart is for all user.
they can suppress the values if they do not want to use . 
user can see the values in the product chart if we do not write them here they need to investigate microservice chart and find out the env vars.